app/models/embedded_ansible_worker.rb

```diff
@@ -18,6 +19,7 @@ def start_runner
       end
     end
   end
+  alias_method :start_runner_via_spawn, :start_runner_via_fork
```

Comment:
I like it too...minor tweak to it:

```
ENV['MIQ_SPAWN_WORKERS'] ? start_runner_via_spawn : start_runner_via_fork
```

You don't need to check for an explicit value in the ENV var, just its presence.